pkg/reconciler/autoscaling/kpa/kpa_test.go

```diff
@@ -457,126 +464,36 @@ func TestReconcile(t *testing.T) {
-			defaultDeployment,
-		}, defaultReady...),
-		WantStatusUpdates: []clientgotesting.UpdateActionImpl{{
-			Object: kpa(testNamespace, testRevision, markActive, markScaleTargetInitialized, WithPAStatusService(testRevision),
-				WithPAMetricsService(privateSvc), withScales(1, defaultScale), WithObservedGeneration(1)),
-		}},
-	}, {
-		Name: "kpa does not become ready without minScale endpoints when reachable",
-		Key:  key,
-		Objects: append([]runtime.Object{
-			kpa(testNamespace, testRevision, withMinScale(2), withScales(1, defaultScale),
-				WithReachabilityReachable, WithPAMetricsService(privateSvc)),
-			defaultSKS,
-			metric(testNamespace, testRevision),
-			defaultDeployment,
-		}, defaultReady...),
-		WantStatusUpdates: []clientgotesting.UpdateActionImpl{{
-			Object: kpa(testNamespace, testRevision, markActivating, withMinScale(2), WithPAMetricsService(privateSvc),
-				withScales(1, defaultScale), WithPAStatusService(testRevision), WithReachabilityReachable,
-				WithObservedGeneration(1)),
-		}},
-	}, {
-		Name: "kpa does not become ready without minScale endpoints when reachability is unknown",
-		Key:  key,
-		Objects: append([]runtime.Object{
-			kpa(testNamespace, testRevision, withMinScale(2), withScales(1, defaultScale),
-				WithPAMetricsService(privateSvc), WithReachabilityUnknown),
-			defaultSKS,
-			metric(testNamespace, testRevision),
-			defaultDeployment,
-		}, defaultReady...),
-		WantStatusUpdates: []clientgotesting.UpdateActionImpl{{
-			Object: kpa(testNamespace, testRevision, markActivating, withMinScale(2), WithPAMetricsService(privateSvc),
-				withScales(1, defaultScale), WithPAStatusService(testRevision), WithReachabilityUnknown,
-				WithObservedGeneration(1)),
-		}},
-	}, {
-		Name: "kpa becomes ready without minScale endpoints when unreachable",
-		Key:  key,
-		Objects: append([]runtime.Object{
-			kpa(testNamespace, testRevision, withMinScale(2), withScales(1, defaultScale),
-				WithPAMetricsService(privateSvc), WithReachabilityUnreachable),
-			defaultSKS,
-			metric(testNamespace, testRevision),
-			defaultDeployment,
-		}, defaultReady...),
-		WantStatusUpdates: []clientgotesting.UpdateActionImpl{{
-			Object: kpa(testNamespace, testRevision, markActive, markScaleTargetInitialized, withMinScale(2), WithPAMetricsService(privateSvc),
-				withScales(1, defaultScale), WithPAStatusService(testRevision), WithReachabilityUnreachable,
-				WithObservedGeneration(1)),
-		}},
-	}, {
-		Name: "kpa becomes ready with minScale endpoints when reachable",
-		Key:  key,
-		Objects: append([]runtime.Object{
-			kpa(testNamespace, testRevision, markActivating, withMinScale(2), WithPAStatusService(testRevision),
-				WithPAMetricsService(privateSvc), withScales(1, defaultScale), WithReachabilityReachable),
-			defaultSKS,
-			metric(testNamespace, testRevision),
-			defaultDeployment,
-		}, makeReadyPods(2, testNamespace, testRevision)...),
-		WantStatusUpdates: []clientgotesting.UpdateActionImpl{{
```

Comment:
Why are these tests deleted? IIUC we only add making dependencies ready/unready operations. It doesn't change other logics.

Reply:
Totally beats me, probably fat typing in vim.